src/Microsoft.PowerShell.Commands.Management/commands/management/TestConnectionCommand.cs

```diff
@@ -479,6 +479,10 @@ private void ProcessTraceroute(string targetNameOrAddress)
         #endregion TracerouteTest
 
         #region MTUSizeTest
+        [System.Diagnostics.CodeAnalysis.SuppressMessage(
+            "Usage",
+            "CA2208:Instantiate argument exceptions correctly",
+            Justification = "https://github.com/PowerShell/PowerShell/issues/13909")]
```

Comment:
The suppression attribute is too broad.
Please use local suppression with directive and c# comment with tracking issue.

Reply:
I would prefer to avoid using preprocessor directives. Is this blocking the PR?

Reply:
Why do you prefer to avoid this?

The attribute will suppress the rule for all current scope but there may be (in the future) other cases that fall under this rule. I think suppression should be more precise - just what is needed.